Classes/Domain/Model/Client.php

```diff
@@ -385,12 +386,23 @@ public function setSla(Sla $sla): void
         $this->sla = $sla;
     }
 
-    public function getTag(): ?Tag
+    /**
+     * Returns the extensions
```

Comment:
Returns the extensions ?

Reply:
I've replaced it with tags, as extensions also have it i will not shorten it here.